libsolidity/codegen/CompilerUtils.cpp

```diff
@@ -1054,6 +1055,7 @@ void CompilerUtils::rightShiftNumberOnStack(unsigned _bits, bool _isSigned)
 unsigned CompilerUtils::prepareMemoryStore(Type const& _type, bool _padToWords)
 {
 	unsigned numBytes = _type.calldataEncodedSize(_padToWords);
+	solAssert(numBytes > 0, "");
 	bool leftAligned = _type.category() == Type::Category::FixedBytes;
 	if (numBytes == 0)
```

Comment:
This is unreachable code.

Reply:
oh :)

Reply:
Also in the other branch these is `number <= 32`. Could be merged into one proper assert up there.

Reply:
This unreachable code is still there.

Reply:
Yes, but now it is reachable.